PythonExample/main.py

```diff
@@ -0,0 +1,62 @@
+from lockstep.lockstep_api import LockstepApi
+import os
+
+
+def retrieve_api_key():
+    API_KEY = os.environ.get('LOCKSTEP_API_SBX')
+    if API_KEY is None:
+        print('NO API KEY')
+    else:
+        print('VALID API KEY FOUND')
+        return API_KEY
+
+
+def create_client(apikey):
+    #     create client
+    env = 'https://api.sbx.lockstep.io/'
+    client = LockstepApi(env)
+    client.with_api_key(apikey)
+    if not client:
+        print("ISSUE WITH CLIENT, NO API KEY OR WRONG ENVIRONMENT")
+    else:
+        print(f"CLIENT WAS CREATED SUCCESSFULLY: {client}")
+        return client
+
+
+def main():
+    API_KEY = retrieve_api_key()
+    client = create_client(API_KEY)
+
+    # Ping
+    status_results = client.status.ping()
+    print(f"StatusResult: {status_results}")
+
+    page_num = 0
+    count = 1
+
+    while page_num < 5: # replace with True, when SDK is working, currently, it is in an endless loop otherwise
+        # Single API call to fetch invoices and company info.
+        # 1/18/2022 Currently SDK query parameters don't work like expected.
```

Comment:
Is this still broken?

Reply:
We can pull the data available to us, but we are not able to grab the Company Name, only the Company Id. I believe it has to do with how our query might be interpreted. If the query is being interpreted correctly then we are missing some data that needs to be made available.